app/lib/audit/models.dart

```diff
@@ -589,21 +592,21 @@ class AuditLogRecord extends db.ExpandoModel<String> {
   ///
   /// Throws [RateLimitException] when the configured rate limit is reached.
   static Future<AuditLogRecord> publisherMemberInviteExpired({
-    required String fromUserId,
+    required String? fromUserId,
     required String publisherId,
     required String memberEmail,
   }) {
     return _checkRateLimit(AuditLogRecord._init()
       ..kind = AuditLogRecordKind.publisherMemberInviteExpired
-      ..agent = fromUserId
+      ..agent = KnownAgents.pubSupport
```

Comment:
I would prefer we make a different alias, we can just invent one.

It's kind of nice that `support@pub.dev` means there was some action by our support team. Some admin action or something.

Maybe for these things we could just write `pub.dev` as the agent name? Because it's the service itself, right?

Reply:
I don't mind either way. The benefit of `support@pub.dev` is that we implicitly communicate the contact address if something is amiss, while in the case of `pub.dev`, while it is short and to the point, needs further search to get to an email. Should I add `pub.dev` to the fixed list of known agents?

Reply:
Note: we already use `support@pub.dev` in `packageRemovedFromPublisher`. Should we also migrate it to `pub.dev`?

Reply:
On a revised note: let's merge this as-is, it improves the state of the audit log records, and we may decide to migrate some of it to `pub.dev` (with the appropriate format verification changes).

Reply:
ship it